frontend/src/locales/scripts/en.json5

```
@@ -52,7 +52,7 @@
     loading: "Loading...",
     filterButton: {
       simple: "Filters",
-      withCount: "{count} Filter|{count} Filters",
+      withCount: "Filters ({count})",
```

@AetherUnbound as per the earlier thread, it'd be good to include a translator note to describe the context of this string, otherwise it might not make sense what's going on here. To do so, you just need to add a JSDoc-style comment on the object property (made possible because json5 supports comments):

```suggestion
      /** Used as the accessible label for the button or tab that opens the filters sidebar. Count refers the number of filters the user has enabled, not the number of available filters */
      withCount: "Filters ({count})",
```

Something like that should be good enough... it still feels obscure if I imagine not knowing the precise context of this, but I'm not sure how detailed or wordy this should be. Maybe worth asking about it after merging in the i18n channel to see if folks have input on how better to handle this for the translators.